SmartDeviceLink/public/SDLLockScreenViewController.h

```diff
@@ -16,6 +16,11 @@ NS_ASSUME_NONNULL_BEGIN
 /// A block that can be used to close the lockscreen when the user swipes on the lockscreen. Override this in your own custom view controllers if you build a custom lock screen.
 typedef void (^SwipeGestureCallbackBlock)(void);
 
+/**
+ *  The SDL Lock Screen's presentation style.
+ */
+@property(nonatomic, assign) UIModalPresentationStyle modalPresentationStyle;
+
```

Comment:
This is not correct. You are not re-defining a property, you are overriding it. Did you test this? I don't think this would work.

```suggestion
```